com.unity.netcode.gameobjects/Runtime/Serialization/NetworkObjectReference.cs

```diff
@@ -139,7 +139,7 @@ public override int GetHashCode()
         /// </summary>
         /// <param name="networkObjectRef">The <see cref="NetworkObjectReference"/> to convert from.</param>
         /// <returns>This returns the <see cref="GameObject"/> that the <see cref="NetworkObject"/> is attached to and is referenced by the <see cref="NetworkObjectReference"/> passed in as a parameter</returns>
-        public static implicit operator GameObject(NetworkObjectReference networkObjectRef) => Resolve(networkObjectRef).gameObject;
+        public static implicit operator GameObject(NetworkObjectReference networkObjectRef) => Resolve(networkObjectRef)?.gameObject;
```

Comment:
Don't use null-conditional operator on MonoBehaviour's - it's not supported. You have to do the full `if (ref != null)` and the engine marshalls to native to verify the object hasn't been destroyed.

Reply:
Changed. Thanks for pointing that out.